### PR TITLE
Update AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_B…

### PR DESCRIPTION
**Pull request by MOC-Labs-Review**
…oards.md

**Replace issue name M00-LAB00:QUICK_DESCRIPTION, for example "M01-LAB01: My new issue" (or same name as linked Issue)**

## Related Issue

**Link related Github Issue** 🢂 Fixes # . (Include issue number after #)

## Checklist 
Mark completed with "x" between brackets, "[x]":
- [ ] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [ ] Tested it
- [ ] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by MOC-Labs-Review** [comment]
Submitted by garjen55 on behalf of MOC Labs Review

File: Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
     > **Note**: Verify that you now see the new stage in the workflow.
 
 1.  Drag the **As a customer, I want to see tutorials I recently viewed** work item from the **Committed** stage into the **QA Approved** stage.
-1.  Drag the **As a customer, I want to view new tutorials** work item from the **QA Approved** stage into the **Done** stage.
+1.  Drag the **As a customer, I want to view new tutorials** work item from the **Approved** stage into the **Done** stage.
```
Comment (unaihuete93) [inline Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:380]: Should be "QA Approved"

Comment (unaihuete93) [reply Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:380]: there is no "Approved Column"

Comment (unaihuete93) [inline Instructions/Labs/AZ400_M01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:380]:
```suggestion
1.  Drag the **As a customer, I want to view new tutorials** work item from the **QA Approved** stage into the **Done** stage.

```

**Review by unaihuete93 — COMMENTED** [review]
please review suggestion

**Comment by unaihuete93** [comment]
@MOC-Labs-Review Next time at least provide description and follow guidelines as mentioned in the PR template if possible Collaboration Guide thanks!

**Comment by unaihuete93** [comment]
imported proposed changes myself (no reply) and merged PR

**Pull request merged**